docs/root/configuration/network_filters/mysql_proxy_filter.rst

```diff
@@ -0,0 +1,46 @@
+.. _config_network_filters_mysql_proxy:
+
+MySQL proxy
+===========
+
+The MySQL proxy filter decodes the wire protocol between the MySQL client
```

Comment:
Is this formatted as 80 chars max? Just curious.

Reply:
Yeah.. emacs.